app/bundles/EmailBundle/Config/config.php

```diff
@@ -307,7 +307,7 @@
                 'serviceAlias' => 'swiftmailer.mailer.transport.%s',
                 'methodCalls'  => [
                     'setUsername'      => ['%mautic.mailer_user%'],
-                    'setPassword'      => ['%mautic.mailer_password%'],
+                    'setPassword'      => ['%mautic.mailer_api_key%'],
```

Comment:
It doesn't seem like it: https://github.com/mautic/mautic/blob/882248a3707a173e58f6a475bdc6c93410b575db/app/bundles/EmailBundle/Form/Type/ConfigType.php#L443